### PR TITLE
Feature/forever loading screen

### PR DESCRIPTION
**Pull request by gustaveen**
Based of https://github.com/HedvigInsurance/web-onboarding/pull/329 so review that one first

Kapture 2020-06-24 at 15 28 46

### CODE REVIEW

File: src/pages/Forever/components/RedeemCode.tsx [hunk 1]
```diff
@@ -1,100 +1,131 @@
...
+    props.disabled ? colorsV3.gray800 : colorsV3.purple500};
 
   @media (min-width: 480px) {
-    width: auto;
+    max-width: ${INPUT_MAX_WIDTH};
+  }
+`
+
+const Info = styled.p`
+  max-width: 25rem;
```
Comment (palmenhq) [inline src/pages/Forever/components/RedeemCode.tsx:95]: `calc(${INPUT_MAX_WIDTH} + 2 * 2rem)` or something maybe makes it clearer how this is connected?

File: src/pages/Forever/components/RedeemCode.tsx [hunk 2]
```diff
@@ -1,100 +1,131 @@
...
-          }}
-        >
-          {({ values }) => (
-            <Form>
+    <Wrapper>
+      <Formik
+        initialValues={{ code }}
+        validationSchema={codeSchema}
+        onSubmit={() => {
+          // Submit
```
Comment (palmenhq) [inline src/pages/Forever/components/RedeemCode.tsx:121]: `TODO Submit` imo

File: src/pages/Forever/components/RedeemCode.tsx [hunk 3]
```diff
@@ -103,24 +134,30 @@ export const RedeemCode: React.FC<RedeemCodeProps> = ({
...
-          background={colorsV3.purple500}
-          foreground={colorsV3.gray900}
-        >
-          {textKeys.FOREVER_LANDINGPAGE_BTN_LABEL()}
-        </SubmitButton>
+        <Info
+          dangerouslySetInnerHTML={{
+            __html: textKeys.FOREVER_LANDINGPAGE_INFO_TEXT(),
+          }}
+        />
```
Comment (palmenhq) [inline src/pages/Forever/components/RedeemCode.tsx:159]: I think rather we could use [`{ MarkdownTranslation } from '@hedviginsurance/textkeyfy'`](https://github.com/HedvigInsurance/web-onboarding/blob/60642f8d28ec70aafa744956dc26ce90d6e5fb5b/src/pages/OfferNew/Checkout/Sign.tsx#L138-L151)

File: src/pages/Forever/index.tsx [hunk 4]
```diff
@@ -12,6 +14,37 @@ type ForeverProps = RouteComponentProps<{
...
+  padding-bottom: 1.15rem;
+
+  @media (min-width: 800px) {
+    justify-content: space-between;
+    padding-top: 2.5rem;
+  }
+`
+
+const LogoLink = styled.a`
+  color: ${colorsV3.gray100};
```
Comment (palmenhq) [inline src/pages/Forever/index.tsx:45]: no hover difference?

File: src/pages/Forever/components/RedeemCode.tsx (hunk 3 above)
Comment (gustaveen) [reply src/pages/Forever/components/RedeemCode.tsx:159]: Ah nice, that's better!

**Comment by gustaveen** [comment]
Updated now @palmenhq

**Pull request merged**